modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -1643,21 +1643,25 @@ public class ApiClient {
      * @return The guessed Content-Type
      */
     public String guessContentTypeFromFile(File file) {
-        String contentType = URLConnection.guessContentTypeFromName(file.getName());
-        if (contentType == null) {
+        try {
+            String contentType = Files.probeContentType(file.toPath());
```

Comment:
shall we use `URLConnection.guessContentTypeFromStream` instead as suggested in https://stackoverflow.com/a/19712111/677735?